app/models/issue/callbacks.rb

```diff
@@ -129,7 +129,7 @@ def calculate_trust_level
     end
 
     def notify_group
-      touch :group_responsibility_notified_at # rubocop:disable Rails/SkipsModelValidations
+      update_attribute :group_responsibility_notified_at, Time.current # rubocop:disable Rails/SkipsModelValidations
```

Comment:
Ziel war auch, dass rubocop:disable zu entfernen:
```suggestion
      self.group_responsibility_notified_at = Time.current
```